lib/goodcity/health_checks/dispatched_packages_order_id_check.rb

```diff
@@ -0,0 +1,19 @@
+require 'goodcity/health_checks/base'
+
+module Goodcity
+  class HealthChecks
+
+    class DispatchedPackagesOrderIdCheck < Base
+      desc "Dispatched packages should contain an order_id reference."
+      def run
+        ids = Package.where(stockit_sent_on: nil, order_id: nil).pluck(:id)
```

Comment:
@steveyken , this query doesn't check for dispatched packages as **stockit_sent_on: nil and order_id: nil** will only give received packages. 
I think **stockit_sent_on should be checked with not nil** here.